astropy/modeling/functional_models.py

```diff
@@ -786,6 +786,53 @@ def inverse(self):
         return self.__class__(slope=new_slope, intercept=new_intercept)
 
 
+class Planar2D(Fittable2DModel):
```

Comment:
I think it's simpler if it does not inherit from `Polynomial2D` which does more computations because it's general.